.github/workflows/Benchmark1Thread.yml

```
@@ -0,0 +1,85 @@
+name: Benchmark (1 Thread)
+
+on:
+  pull_request:
+
+permissions:
+  pull-requests: write
+
+jobs:
+  benchmark-1-thread:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v5
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          persist-credentials: false
+
+      - name: Setup Julia
+        uses: julia-actions/setup-julia@v2
+        with:
+          version: '1.11'
```

We should probably test the latest version as well
```suggestion
          version: '1', '1.11'
```